lib/Elastica/Query/GeohashCell.php

```diff
@@ -1,10 +1,12 @@
 <?php
 namespace Elastica\Query;
 
+trigger_error('Elastica\Query\GeohashCell is deprecated.', E_USER_DEPRECATED);
+
 /**
  * Class GeohashCell.
  *
- * @link https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-geohash-cell-query.html
+ * @link https://www.elastic.co/guide/en/elasticsearch/reference/2.4/query-dsl-geohash-cell-query.html
```

Comment:
This was removed in 5.0? If yes, we can also remove it here as it not work with 5.0 anyways, correct? But we should mention it in the changelog.